build.sbt

```diff
@@ -943,7 +943,7 @@ def genSmithyImpl(config: Configuration) = Def.task {
       .map(_.data)
 
   val mc = "smithy4s.codegen.cli.Main"
-  val s = streams.value
+  val s = (config / streams).value
```

Comment:
for future generations: this is our local version of the fix in #502 that I forgot to port here.